plugins/generator-1.19.2/forge-1.19.2/procedures/entity_stop_navigation.java.ftl

```diff
@@ -1,2 +1,2 @@
-if (${input$entity} instanceof Mob _entity)
-	_entity.getNavigation().stop();
\ No newline at end of file
+if (${input$entity} instanceof Mob _livEnt)

```

Comment:
Renames like this one are not needed, they are only needed for output blocks. As long as output blocks only use _livEnt and statement blocks _entity, those renames are not needed and relevant to the #3613

Reply:
This is why I stressed we just need to make sure to request proper names of blocks when checking new procedure PRs. The issue was in fact caused by wrong namings of some of procedure blocks added in 2023.1 snapshots

Reply:
`entity_pathfind_to` is procedural block type and uses `_entity` name already.

Reply:
How can one procedural block get in the code of another? This is exact thing I said in https://github.com/MCreator/MCreator/issues/3613#issuecomment-1453115758 and what @Goldorion confirmed in https://github.com/MCreator/MCreator/issues/3613#issuecomment-1453922459 and what I stressed in reply here:

> Renames like this one are not needed, they are only needed for output blocks.

Reply:
You said
>output blocks only use _livEnt and statement blocks _entity

which did happen to `entity_pathfind_to` before my changes here. What do I miss?

Reply:
Sorry, I forgot to edit this part of my comment. I have deleted it now, but here it is for the reference:

> Output blocks that do not use inner classes:
> 
> * entity_pathfind_to: _entity to _livEnt (find in all files for _livEnt shows that most blocks solve this by using _livEnt)

This statement was wrong from my side as pathfind to is not even output block 🤦

What is true is this:

> Renames like this one are not needed, they are only needed for output blocks. As long as output blocks only use _livEnt and statement blocks _entity, those renames are not needed and relevant to the https://github.com/MCreator/MCreator/issues/3613

Reply:
In other words: statement block can't not appear inside another statement block, so taking care of variable name collisions is not needed between them.

However, output blocks can appear in statement blocks, so local variable names in output blocks must not collide with local variable names of statement blocks.

In same spirit, statement blocks can appear in statement blocks with statement inputs. So local variable names in statement blocks with statement inputs must not collide with local variables of statement blocks (without statement inputs).

Common group is statement blocks, so we keep them as they are, but take care that:

* output blocks
* statement blocks with statement inputs

use local variable names not used by statement blocks.

Thus changes are needed (where they are needed for)

* output blocks
* statement blocks with statement inputs